### PR TITLE
spring-data-mongodb: initial integration

### PR DESCRIPTION
**Pull request by onionpsy**

### CODE REVIEW

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: projects/spring-data-mongodb/project.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+homepage: "https://github.com/spring-projects/spring-data-mongodb"
+language: jvm
+main_repo: "https://github.com/spring-projects/spring-data-mongodb.git"
+primary_contact: "pwebb@vmware.com"
+fuzzing_engines:
+  - libfuzzer
+sanitizers:
+  - address
+vendor_ccs:
+  - "wilkinsona@vmware.com"
```
Comment (jonathanmetzman) [inline projects/spring-data-mongodb/project.yaml:10]: Shouldn't this person be under auto_cc, as they are a project maintainer, not a vendor?

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]

File: projects/spring-data-mongodb/project.yaml (hunk 1 above)
Comment (onionpsy) [reply projects/spring-data-mongodb/project.yaml:10]: Yes, sorry. I also changed it for https://github.com/google/oss-fuzz/pull/8764

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**